test/e2e/all_in_one_test.go

```diff
@@ -256,6 +256,26 @@ func TestDeployAllInOneEnterprisePostgres(t *testing.T) {
 	verifyEnterpriseWithPostgres(ctx, t, env, adminPassword)
 }
 
+func TestDeployAllInOnePostgresGatewayDiscovery(t *testing.T) {
+	t.Parallel()
+
+	const (
+		manifestFileName = "all-in-one-postgres-multiple-gateways.yaml"
+		manifestFilePath = "../../deploy/single/" + manifestFileName
+	)
```

Comment:
```suggestion
	const manifestFilePath = "manifests/all-in-one-postgres-multiple-gateways.yaml"
```
Script for generating and path was changed in 
- https://github.com/Kong/kubernetes-ingress-controller/pull/4970

hence please regenerate manifests and remove `deploy/single/all-in-one-postgres-multiple-gateways.yaml`